src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+part of the user experience when the user decides to interact with a site that has just been loaded.
+FID captures some of the time that the user has to wait in order to see a response from their
+interaction with a site. In other words, FID is a lower bound on the amount of time a user waits
+after interacting.
+
+Other metrics like [Total Blocking Time (TBT)](/tbt) and [Time To Interactive (TTI)](/tti) are based
+on [long tasks](https://developer.mozilla.org/en-US/docs/Web/API/Long_Tasks_API) and, like FID, also
+measure main thread blocking time during load. Since these metrics can be measured in both the field
+and the lab, many developers have asked why we don't prefer one of these over FID.
+
+There are several reasons for this. Perhaps the most important reason is that these metrics do not
+measure the user experience directly. All of these metrics measure how much JavaScript runs on the
+page. While long running JavaScript does tend to cause problems to sites, these tasks don’t
+necessarily impact the user experience if the user is not interacting with the page when they occur.
+A page can have a great score on these metrics and feel slow or it can have a poor score while
+feeling fast for users. In our experience, these indirect measurements result in metrics that work
+great for some sites but not for most sites. In short, the fact that long tasks and TTI are not
+user-centric makes these weaker candidates.
+
+While [lab measurement](/user-centric-performance-metrics/#in-the-lab) is certainly important and an
+invaluable tool for diagnostics, what really matters is how users experience sites. By having a
+user-centric metric that reflects real-user conditions, you are guaranteed to capture something
+meaningful about the experience. We decided to start with a small portion of that experience, even
+though we know this portion is not representative of the full experience. This is why we’re working
+on capturing a larger chunk of the time a user waits for their inputs to be handled.
+
+{% Aside %}If you’re interested in a deeper dive into some of the metrics we looked into, here is a
+[study about
+TBT](https://docs.google.com/document/d/1xCERB_X7PiP5RAZDwyIkODnIXoBk-Oo7Mi9266aEdGg/edit#heading=h.ypzsa9g2mv2g)
+and a [study about
+TTI](https://docs.google.com/document/d/1sHy6R58olikMTwk5hkJL4jd9S1jbksdMY5ve3Shdg-g/edit). {%
+endAside %}
+
+{% Details %}
+
+{% DetailsSummary %}
+A note on measuring TTI in the field
+{% endDetailsSummary %}
+
+Measuring TTI on real users in the field is problematic because it occurs very late in the page
+load. A 5-second network quiet window is required before TTI can even be computed. In the lab, you
+can choose to unload the page whenever you have all the data that you need, but that’s not the case
+with real-user monitoring in the field. A user may choose to leave the page or interact with it at
+any time. In particular, users may choose to leave pages that take a long time to load, and TTI will
```

Comment:
Nit: `and TTI will not be recorded` --> `and accurate TTI will cannot be recorded`.